makisu/README.md

```diff
@@ -0,0 +1,63 @@
+# makisu
+
+This build template builds source into a container image using uber's
+[`makisu`](https://github.com/uber/makisu) tool.
+
+>Makisu is a fast and flexible Docker image build tool designed for unprivileged
+>containerized environments such as Mesos or Kubernetes.
+> - [makisu website](https://github.com/uber/makisu)
+
+makisu is meant to be run as an image, `gcr.io/makisu-project/makisu`. This
+makes it a perfect tool to be part of a Knative build.
+
+## Create the registry configuration
+
+makisu uses a [registry configuration](https://github.com/uber/makisu/blob/master/docs/REGISTRY.md) which should be stored as a secret in Kubernetes. Adjust the `registry.yaml` in this diretroy to contain your user and password for the Docker hub (or configure a different [registry](https://github.com/uber/makisu/blob/master/docs/REGISTRY.md#examples)). Keep in mind that the secret must exist in the same namespace as the build runs.:
+
+```bash
+kubectl --namespace default create secret generic docker-registry-config --from-file=./registry.yaml
+```
+
+## Create the template
+
+```
+kubectl apply -f https://raw.githubusercontent.com/knative/build-templates/master/makisu/makisu.yaml
+```
+
+## Parameters
+
+* **IMAGE**: The Docker image name to apply to the newly built image.
+  (_required_)
+* **CONTEXTPATH**: The path to the build context (_default:_
+  `/workspace`)
+* **PUSH_REGISTRY**: The Registry to push the image to (_default:_
+  `index.docker.io`)
+* **REGISTRY_SECRET**: Secret containing information about the used regsitry (_default:_
+  `docker-registry-config`) **This Parameter currently doesn't work: https://github.com/knative/build/pull/550**
```

Comment:
This PR is merged now, available in the latest Knative Build release (0.4)

Reply:
I adjusted it

Reply:
It looks like this still says it doesn't work?

Reply:
Good catch, thanks!